src/app/core.ts

```diff
@@ -1,40 +1,20 @@
 import {messages} from '../constants/messages';
 import {client} from '../helpers/client/client';
-import {
-  errorLog,
-  generateErrorBody,
-  generatePromiseObjects,
-  isValidTransferSpec,
-  randomUUID,
-  throwError
-} from '../helpers/helpers';
+import {errorLog, generateErrorBody, generatePromiseObjects, isValidTransferSpec, randomUUID, throwError} from '../helpers/helpers';
 import {asperaBrowser} from '../index';
-import {DesktopInfo, TransferResponse} from '../models/aspera-browser.model';
-import {
-  CustomBrandingOptions,
-  DataTransferResponse,
-  BrowserSpec,
-  BrowserStyleFile,
-  BrowserTransfer,
-  FileDialogOptions,
-  FolderDialogOptions,
-  ModifyTransferOptions,
-  ResumeTransferOptions,
-  SafariExtensionEvents,
-  TransferSpec,
-  WebsocketEvents
-} from '../models/models';
+import {BrowserInfo, TransferResponse} from '../models/aspera-browser.model';
+import {CustomBrandingOptions, DataTransferResponse, BrowserSpec, BrowserStyleFile, BrowserTransfer, FileDialogOptions, FolderDialogOptions, InitOptions, ModifyTransferOptions, ResumeTransferOptions, SafariExtensionEvents, TransferSpec, WebsocketEvents} from '../models/models';
 
 /**
- * Check if IBM Aspera Browser connection works. This function is called by init
+ * Check if IBM Aspera for Desktop connection works. This function is called by init
  * when initializing the SDK. This function can be used at any point for checking.
  *
  * @returns a promise that resolves if server can connect or rejects if not
  */
-export const testBrowserConnection = (): Promise<any> => {
```

Comment:
This was a breaking change.  Aspera Starter was using `testBrowserConnection`

Reply:
Yeah but we haven't deployed the new SDK yet.

Are you using the latest release? If so, we can change it in Aspera Starter after deploying the new version.

Reply:
Yes; it was deployed to fix a spec change.  However, seeing other issues now with init() call.

Reply:
Oh that makes sense, sorry I didn't know it was already deployed.